.docker/entry-point.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+set -e
+
+# Set the apache user and group to match the host user.
+OWNER=$(stat -c '%u' /data)
+GROUP=$(stat -c '%g' /data)
+if [ "$OWNER" != "0" ]; then
```

Comment:
Should the existance of GROUP also be checked? Right now you're only deleting the pktool user if it exists but not the group.